readme/Index.scalatex

```diff
@@ -368,6 +368,29 @@
     @p
       The @hl.scala{debug()} call returns @hl.scala{: Any}; by default, this is @hl.scala{(): Unit}, but you can also return custom values by passing in an argument to @hl.scala{exit(...)} when you exit the REPL. This value will then be returned from @hl.scala{debug()}, and can be used in the rest of your Scala application.
 
+    @p
+      Ammonite can also be used to remotely connect to your running application and interact with it in real-time, similar to erlang `erl -remsh` command. To do this, add ammonite sshd server to your classpath, for example with SBT:
+
+    @hl.scala
+      libraryDependencies += "com.lihaoyi" % "ammonite-repl-sshd" % "@ammonite.Constants.version" cross CrossVersion.full
+
+    @p
+      Now add repl server to your application:
+
+    @hl.scala
+      import ammonite.repl.sshd._
+      val replServer = new SshdRepl(SshServerConfig(port = 22222, username = "repl", password = "your_secure_password"))
```

Comment:
Does it currently support key-based authentication? If so we should describe how. If not, we should also mention it =)